experimenter/experimenter/features/__init__.py

```diff
@@ -80,30 +82,53 @@ def get_jsonschema(self):
 
 
 class Features:
-    _features = None
+    _features: Optional[list[Feature]] = None
```

Comment:
Do we want this to be `list[Feature]` instead of Optional?

Reply:
We want it to be Optional because we're assigning it to None right there :)